huxley/www/js/components/AdvisorView.js

```diff
@@ -32,21 +33,23 @@ var AdvisorView = React.createClass ({
     return (
       <div>
         <TopBar user={this.props.user} />
-        <div className="navbar rounded-top">
-          <NavTab href="/advisor/profile">
-            Profile
-          </NavTab>
-          <NavTab href="/advisor/assignments">
-            Assignments
-          </NavTab>
-          <NavTab href="/advisor/roster">
-            Delegates
-          </NavTab>
-          <NavTab href="/advisor/feedback">
-            Feedback
-          </NavTab>
-        </div>
-        {this.props.children}
+        <Shaker>
```

Comment:
Probably. I haven't thought too much about it; I figure we'll cross that bridge when we get to it. One option might be renaming this component to be `Animatable` or something and have it support whichever animations we want to execute on it.